tutorials/workzone-enrich-5-integrate-card/workzone-enrich-5-integrate-card.md

```diff
@@ -47,7 +47,25 @@ The card is now enabled and can be added to SAP Work Zone pages.
 [DONE]
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 2: ](Add your UI5 integration card to your SAP Work Zone workspace)]
+[ACCORDION-BEGIN [Step 2: ](Configure the card settings )]
+
+Before we add the card to our workspace, let's configure one of the settings of the card.
```

Comment:
When addressing the use, please don't say "we", say "you" instead.
There are two more places where you are using the "royal we". Please change them accordingly.

Reply:
Done